services/app-api/prisma/schema.prisma

```diff
@@ -130,21 +131,21 @@ model RateRevisionTable {
   submitInfoID String?
   submitInfo   UpdateInfoTable? @relation("submitRateInfo", fields: [submitInfoID], references: [id], onDelete: Cascade)
 
-  rateType                       RateType?
-  rateCapitationType             RateCapitationType?
-  rateDocuments                  RateDocument[]
-  supportingDocuments            RateSupportingDocument[] 
-  rateDateStart                  DateTime?                  @db.Date
-  rateDateEnd                    DateTime?                  @db.Date
-  rateDateCertified              DateTime?                  @db.Date
-  amendmentEffectiveDateStart    DateTime?                  @db.Date
-  amendmentEffectiveDateEnd      DateTime?                  @db.Date
-  rateProgramIDs                 String[]
-  rateCertificationName          String?
-  certifyingActuaryContacts      ActuaryContact[]           @relation(name: "CertifyingActuaryOnRateRevision")
-  addtlActuaryContacts           ActuaryContact[]           @relation(name: "AddtlActuaryOnRateRevision")
-  actuaryCommunicationPreference ActuaryCommunication?
-  contractsWithSharedRateRevision ContractTable[] @relation(name: "SharedRateRevisions")
+  rateType                        RateType?
+  rateCapitationType              RateCapitationType?
+  rateDocuments                   RateDocument[]
+  supportingDocuments             RateSupportingDocument[]
+  rateDateStart                   DateTime?                @db.Date
+  rateDateEnd                     DateTime?                @db.Date
+  rateDateCertified               DateTime?                @db.Date
+  amendmentEffectiveDateStart     DateTime?                @db.Date
+  amendmentEffectiveDateEnd       DateTime?                @db.Date
+  rateProgramIDs                  String[]
```

Comment:
Hmm maybe we need to build in prisma formatting into our precommit scripts. I've been noticing these kind of formatting difff on our prisma PRs over the last few months